testsuite/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~  Copyright 2018 Red Hat, Inc.
+ ~
+ ~  Licensed under the Apache License, Version 2.0 (the "License");
+ ~  you may not use this file except in compliance with the License.
+ ~  You may obtain a copy of the License at
+ ~
+ ~    http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~  Unless required by applicable law or agreed to in writing, software
+ ~  distributed under the License is distributed on an "AS IS" BASIS,
+ ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~  See the License for the specific language governing permissions and
+ ~  limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.smallrye</groupId>
+    <artifactId>smallrye-jwt-parent-1.1</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
+  </parent>
+
+  <packaging>pom</packaging>
+  <artifactId>smallrye-jwt-testsuite-parent-1.1</artifactId>
+  <name>SmallRye: MicroProfile JWT - Testsuite Parent</name>
+
+  <modules>
+    <module>basic</module>
```

Comment:
This needs plugin configuration the same as https://github.com/smallrye/smallrye-config/blob/master/testsuite/pom.xml#L38 to prevent any testsuite from being released to Nexus

Reply:
@kenfinnigan have a look please, also removed some basic/pom.xml dependencies which are not used